### PR TITLE
Fix for oze ENT using /v1/config/Pronote

### PR DESCRIPTION
**Pull request by johan-perso**
This fix allow to login even with school that doesn't put the proxySSO in ozapps

I have done all the following:

- [x] added the feature
- [ ] added documentation for the feature
- [ ] added tests for the feature
- [x] ran mypy, black, and unit tests

### CODE REVIEW

File: pronotepy/ent/generic_func.py [hunk 1]
```diff
@@ -412,8 +412,17 @@ def _oze_ent(
             if app["code"] == "pronote":
                 proxysso_url = urljoin(url, app["externalRoute"])
 
+        # If we still haven't got the url, try something else
```
Comment (bain3) [inline pronotepy/ent/generic_func.py:415]: Where is the if statement checking if we already have the URL or not?

**Review by bain3 — CHANGES_REQUESTED** [review]
Hello!
Thank you for your contribution. Could you please run black formatter on the file?

File: pronotepy/ent/generic_func.py [hunk 2]
```diff
@@ -412,8 +412,17 @@ def _oze_ent(
...
+        pronoteConfig_url = urljoin(api_url, "/v1/config/Pronote")
+        payload = {}
+        payload["ctx_profil"] = ctx_profil
+        payload["ctx_etab"] = ctx_etab
```
[the remark this answers is not in the document]
Comment (johan-perso) [reply pronotepy/ent/generic_func.py:419]: Oh yes I didn't think about it, I'm not even learning Python so I'm not the best developer, sorry

File: pronotepy/ent/generic_func.py (hunk 1 above)
Comment (johan-perso) [reply pronotepy/ent/generic_func.py:415]: I'm gonna do a quick new commit to fix this, while running black formatter

**Review by bain3 — APPROVED** [review]
LGTM!

**Pull request merged**